### ISSUE TITLE
Error when going from 'Analysis' to 'Cell viewer'

### ISSUE THREAD

**Issue by iezqrom** [bkntr/napari-brainways#25]
After we detect cells and we go to 'Analysis', then if we go back to 'Cell Viewer' we get the error below. It happens in both Linux and Windows. The only solution is to restart the software.

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
File ~/miniconda3/envs/microscoping/lib/python3.10/site-packages/napari_brainways/brainways_ui.py:236, in BrainwaysUI._open_step(self=<napari_brainways.brainways_ui.BrainwaysUI object>)
    234 def _open_step(self):
    235     self.current_step.open()
--> 236     self.current_step.show(self.current_params, self._image)
        self._image = array([[4, 3, ..., 7, 7],
       [6, 5, ..., 4, 4],
       ...,
       [9, 8, ..., 4, 2],
       [8, 8, ..., 3, 2]], dtype=uint8)
        self = <napari_brainways.brainways_ui.BrainwaysUI object at 0x7d8c04241090>
    237     self.widget.update_enabled_steps()
    238     self._set_title()

File ~/miniconda3/envs/microscoping/lib/python3.10/site-packages/napari_brainways/controllers/cell_3d_viewer_controller.py:88, in Cell3DViewerController.show(self=<napari_brainways.controllers.cell_3d_viewer_controller.Cell3DViewerController object>, params=BrainwaysParams(atlas=AtlasRegistrationParams(ap....18045043945312, 135.1804656982422]]), cell=None), image=array([[4, 3, ..., 7, 7],
       [6, 5, ..., 4, ...., 4, 2],
       [8, 8, ..., 3, 2]], dtype=uint8), from_ui=False)
     86     self.show_3d()
     87 else:
---> 88     self.show_2d(image=self._image, from_ui=from_ui)
        self = <napari_brainways.controllers.cell_3d_viewer_controller.Cell3DViewerController object at 0x7d8bfc2e9810>
        self._image = array([[4, 3, ..., 7, 7],
       [6, 5, ..., 4, 4],
       ...,
       [9, 8, ..., 4, 2],
       [8, 8, ..., 3, 2]], dtype=uint8)
        from_ui = False

File ~/miniconda3/envs/microscoping/lib/python3.10/site-packages/napari_brainways/controllers/cell_3d_viewer_controller.py:134, in Cell3DViewerController.show_2d(self=<napari_brainways.controllers.cell_3d_viewer_controller.Cell3DViewerController object>, image=array([[4, 3, ..., 7, 7],
       [6, 5, ..., 4, ...., 4, 2],
       [8, 8, ..., 3, 2]], dtype=uint8), from_ui=False)
    132     self.atlas_layer = None
    133 if self.input_layer is not None:
--> 134     self.ui.viewer.layers.remove(self.input_layer)
        self.input_layer = <Image layer 'Image' at 0x7d8bd4574f40>
        self = <napari_brainways.controllers.cell_3d_viewer_controller.Cell3DViewerController object at 0x7d8bfc2e9810>
        self.ui = <napari_brainways.brainways_ui.BrainwaysUI object at 0x7d8c04241090>
        self.ui.viewer = Viewer(camera=Camera(center=(255.5, 255.5, 255.5), zoom=1.7051757812499997, angles=(0.0, 0.0, 89.99999999999999), perspective=0.0, mouse_pan=True, mouse_zoom=True), cursor=Cursor(position=(255.5, 485.9747813831676, 405.92437969950777), scaled=True, style=<CursorStyle.STANDARD: 'standard'>, size=1.0), dims=Dims(ndim=3, ndisplay=3, order=(0, 1, 2), axis_labels=('0', '1', '2'), rollable=(True, True, True), range=(RangeTuple(start=-0.5, stop=511.5, step=1.0), RangeTuple(start=-0.5, stop=511.5, step=1.0), RangeTuple(start=-0.5, stop=511.5, step=1.0)), margin_left=(0.0, 0.0, 0.0), margin_right=(0.0, 0.0, 0.0), point=(255.5, 255.5, 255.5), last_used=0), grid=GridCanvas(stride=1, shape=(-1, -1), enabled=False), layers=[<Points layer 'Detected Cells' at 0x7d8bbd5aa920>], help='use <5> for transform, use <2> for add points, use <3> for select points', status={'layer_name': 'Detected Cells', 'layer_base': 'Detected Cells', 'source_type': '', 'plugin': '', 'coordinates': ' [256 486 406]'}, tooltip=Tooltip(visible=False, text=''), theme='dark', title='test2 - /home/iezquer/Documents/data_server/wetness/wetness_brainwide_immuno_cfos/data/raw/AFQ0_AT107_20240802/Process_863.vsi [Scene #0]', mouse_over_canvas=True, mouse_move_callbacks=[], mouse_drag_callbacks=[], mouse_double_click_callbacks=[], mouse_wheel_callbacks=[<function dims_scroll at 0x7d8c09c3a290>], _persisted_mouse_event={}, _mouse_drag_gen={}, _mouse_wheel_gen={}, _keymap={<KeyBinding at 0x7d8bfc1f78e0: PageDown>: <bound method BrainwaysUI.next_step of <napari_brainways.brainways_ui.BrainwaysUI object at 0x7d8c04241090>>, <KeyBinding at 0x7d8bfc1f7d00: PageUp>: <bound method BrainwaysUI.prev_step of <napari_brainways.brainways_ui.BrainwaysUI object at 0x7d8c04241090>>, <KeyBinding at 0x7d8bfc1f7c40: N>: <bound method BrainwaysUI.next_image of <napari_brainways.brainways_ui.BrainwaysUI object at 0x7d8c04241090>>, <KeyBinding at 0x7d8bfc1f7d90: B>: <bound method BrainwaysUI.prev_image of <napari_brainways.brainways_ui.BrainwaysUI object at 0x7d8c04241090>>, <KeyBinding at 0x7d8be218fb80: Shift+N>: <bound method BrainwaysUI.next_subject of <napari_brainways.brainways_ui.BrainwaysUI object at 0x7d8c04241090>>, <KeyBinding at 0x7d8be218d840: Shift+B>: <bound method BrainwaysUI.prev_subject of <napari_brainways.brainways_ui.BrainwaysUI object at 0x7d8c04241090>>, <KeyBinding at 0x7d8be218e680: Home>: <function BrainwaysUI._register_keybinds.<locals>.<lambda> at 0x7d8be21a3f40>, <KeyBinding at 0x7d8be027c700: End>: <function BrainwaysUI._register_keybinds.<locals>.<lambda> at 0x7d8be21a3eb0>, <KeyBinding at 0x7d8bbeb9cd00: Ctrl+Down>: <function RegistrationController.get_keybind_fn.<locals>._fn at 0x7d8bd5755ab0>, <KeyBinding at 0x7d8bbeb9c100: Ctrl+Up>: <function RegistrationController.get_keybind_fn.<locals>._fn at 0x7d8bd5755000>, <KeyBinding at 0x7d8c04dc2e90: Left>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca3c70>, <KeyBinding at 0x7d8bd570c910: Right>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca2f80>, <KeyBinding at 0x7d8bd4d53b20: Up>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca1900>, <KeyBinding at 0x7d8c06482950: Down>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca2e60>, <KeyBinding at 0x7d8c064827d0: Shift+Left>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca30a0>, <KeyBinding at 0x7d8c06483fa0: Shift+Right>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca2cb0>, <KeyBinding at 0x7d8c062b6e60: Shift+Up>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca15a0>, <KeyBinding at 0x7d8c06480490: Shift+Down>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca11b0>, <KeyBinding at 0x7d8c06480850: Ctrl+Left>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca12d0>, <KeyBinding at 0x7d8bbffa08e0: Ctrl+Right>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca2d40>, <KeyBinding at 0x7d8bbf5b2aa0: Alt+Left>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca1510>, <KeyBinding at 0x7d8be0113910: Alt+Right>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca08b0>, <KeyBinding at 0x7d8bd61069e0: Alt+Up>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca0820>, <KeyBinding at 0x7d8bd6104970: Alt+Down>: <function Affine2DController.keybind_modify_params.<locals>._func at 0x7d8c04ca0670>, <KeyBinding at 0x7d8bd4912ce0: Ctrl+Z>: <bound method TpsController.previous_params of <napari_brainways.controllers.tps_controller.TpsController object at 0x7d8c0423c7f0>>, <KeyBinding at 0x7d8bd4910670: Ctrl+Y>: <bound method TpsController.next_params of <napari_brainways.controllers.tps_controller.TpsController object at 0x7d8c0423c7f0>>, <KeyBinding at 0x7d8bd4911f60: Ctrl+Shift+Z>: <bound method TpsController.next_params of <napari_brainways.controllers.tps_controller.TpsController object at 0x7d8c0423c7f0>>, <KeyBinding at 0x7d8bd4910bb0: A>: <bound method TpsController.set_points_mode_add of <napari_brainways.controllers.tps_controller.TpsController object at 0x7d8c0423c7f0>>, <KeyBinding at 0x7d8bd61e6470: >: <bound method TpsController.show_help of <napari_brainways.controllers.tps_controller.TpsController object at 0x7d8c0423c7f0>>})
    135 self.input_layer = self.ui.viewer.add_image(image, name="Image")
    136 update_layer_contrast_limits(self.input_layer)

File ~/miniconda3/envs/microscoping/lib/python3.10/site-packages/napari/utils/_proxies.py:194, in CallablePublicOnlyProxy.__call__(self=<bound method MutableSequence.remove of [<Points layer 'Detected Cells' at 0x7d8bbd5aa920>]>, *args=(<Image layer 'Image'>,), **kwargs={})
    186 args = tuple(
    187     arg.__wrapped__ if isinstance(arg, PublicOnlyProxy) else arg
    188     for arg in args
    189 )
    190 kwargs = {
    191     k: v.__wrapped__ if isinstance(v, PublicOnlyProxy) else v
    192     for k, v in kwargs.items()
    193 }
--> 194 return self.create(self.__wrapped__(*args, **kwargs))
        args = (<Image layer 'Image' at 0x7d8bd4574f40>,)
        kwargs = {}
        self = <bound method MutableSequence.remove of [<Points layer 'Detected Cells' at 0x7d8bbd5aa920>]>

File ~/miniconda3/envs/microscoping/lib/python3.10/_collections_abc.py:1163, in MutableSequence.remove(self=[<Points layer 'Detected Cells' at 0x7d8bbd5aa920>], value=<Image layer 'Image'>)
   1159 def remove(self, value):
   1160     '''S.remove(value) -- remove first occurrence of value.
   1161        Raise ValueError if the value is not present.
   1162     '''
-> 1163     del self[self.index(value)]
        self = [<Points layer 'Detected Cells' at 0x7d8bbd5aa920>]
        value = <Image layer 'Image' at 0x7d8bd4574f40>

File ~/miniconda3/envs/microscoping/lib/python3.10/site-packages/napari/utils/events/containers/_typed.py:236, in TypedMutableSequence.index(self=[<Points layer 'Detected Cells' at 0x7d8bbd5aa920>], value=<Image layer 'Image'>, start=0, stop=None)
    233     if v is value or v == value:
    234         return i
--> 236 raise ValueError(
        trans = <napari.utils.translations.TranslationBundle object at 0x7d8c165f5210>
        value = <Image layer 'Image' at 0x7d8bd4574f40>
    237     trans._(
    238         '{value!r} is not in list',
    239         deferred=True,
    240         value=value,
    241     )
    242 )

ValueError: <Image layer 'Image' at 0x7d8bd4574f40> is not in list
```

**Comment by bkntr** [comment]
This bug should be fixed in the latest version v0.1.13.1, please try to update with the command:
`pip install -U napari-brainways`

**Comment by iezqrom** [comment]
Yes! I didn't get this error
